### ISSUE TITLE
Binding settings not translated in Main UI

### ISSUE THREAD

**Issue by lolodomo** [openhab/openhab-webui#2641]
## The problem

Main UI presents untranslated label and description for binding settings while the translation is present in the properties file.

It concerns certain bindings like netatmo or sonos for example.

![image](https://github.com/openhab/openhab-webui/assets/10186704/a6a8e3fe-a089-44db-8036-531fa8056121)

It is ok for addons like openhabcloud or voiceRSS.

![image](https://github.com/openhab/openhab-webui/assets/10186704/ed9213ca-e45e-4d09-938f-012fa34a83f9)

## Expected behavior

Main UI should consider existing translations for binding settings.

## Additional information

I am asking myself if the problem would be when the addon.xml file contains config-description tag (and so all config settings declared in the addon.xml file) instead of a config-description-ref tag (config settings declared in a separate config.xml file) ?

Or maybe the translations are considered for voice and io addons but not binding addons ?

**Comment by lolodomo** [comment]
I can see that Main UI is requesting /rest/config-descriptions/binding:netatmo to retrieve the binding parameters and the response is not localized. So the problem is in core framework.

@wborn @jlaur : maybe there is a mismatch between "binding" and "addon" ? In our properties files, the entries are:
```
addon.config.netatmo.readFriends.label = Access Guests
addon.config.netatmo.readFriends.description = For Weather Stations: A friend gave you access to their Netatmo Weather Station.
```

**Comment by florian-h05** [comment]
> So the problem is in core framework.

Exactly - Main UI only displays what it gets from the REST API.

**Comment by lolodomo** [comment]
As I suspected, the core framework is inferring a key starting by "binding" and not "addon".
https://github.com/openhab/openhab-core/blob/918b4faa3b3254a993774147f798e8bc59580bf6/bundles/org.openhab.core.config.core/src/main/java/org/openhab/core/config/core/internal/i18n/ConfigDescriptionI18nUtil.java#L94

I really do not know why we used "addon" in all bindings properties.

To avoid changing all bindings, I will propose a fix in the `inferKey` method to use "binding" instead of "addon".
Already tested and working.
